spec/spec_helper.rb

```diff
@@ -49,10 +49,12 @@ def params_wrapper(args)
 require "ffaker"
 
 # Migrate
+# First remove any dummy or other stray migrations built from `rails g update_sti`
 require_relative "support/paper_trail_spec_migrator"
-::PaperTrailSpecMigrator.
-  new(::File.expand_path("dummy_app/db/migrate/", __dir__)).
-  migrate
+migrator = ::PaperTrailSpecMigrator.new
+migrator.delete("*_dummy_migration.rb")
+migrator.delete("*_update_versions_for_sti.rb")
+migrator.migrate
```

Comment:
Would it be cleaner to move the deletions into the migrator class? Would we ever want to migrate without deleting them?

Reply:
If my new generator tests fail in somewhat interesting ways then stray files can be left in db/migrate, and I want to make absolutely sure that nothing extra is left around.

Reply:
Sorry, I don't understand. Why can't `PaperTrailSpecMigrator#migrate` perform deletion?

Reply:
Well, `#migrate` certainly does, but in addition I like to make sure of things.  Picture a case where an example for a generator is interrupted or times out.  I want there to be no risks in this scenario.